bokeh/models/annotations.py

```diff
@@ -220,3 +222,21 @@ class Span(Annotation):
     line_props = Include(LineProps, use_prefix=False, help="""
     The %s values for the span.
     """)
+
+class Tooltip(Annotation):
+    """ Render a tooltip.
+
+    .. note::
+        This model is typically managed by BokehJS and is not useful
+        directly from python.
+
```

Comment:
it isn't clear to me what this note means - does it mean setting the properties on this won't work?
or just that I probably don't want to set them?